### ISSUE TITLE
Kimai Version 1.23.0 does not contain proper version number / does not update properly

### ISSUE THREAD

**Issue by de-sec** [kevinpapst/kimai2#3498]
**Describe the bug**
Kimai Version 1.23.0 does not contain proper version number / does not update properly

**To Reproduce**
update to 1.23.0

**Logfile**

```
# git fetch --tags
remote: Enumerating objects: 523, done.
remote: Counting objects: 100% (203/203), done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 63 (delta 53), reused 30 (delta 26), pack-reused 0
Unpacking objects: 100% (63/63), 43.51 KiB | 730.00 KiB/s, done.
From https://github.com/kevinpapst/kimai2
 * [new tag]           1.23.0     -> 1.23.0


# git checkout 1.23.0
Previous HEAD position was 303ec5be bump composer packages for version 1.22.1 (#3455)
HEAD is now at ad20fd1c Translated using Weblate (#3482)
[root@ze kimai2]# composer install --no-dev --optimize-autoloader -n
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
**Nothing to install, update or remove**
Package symfony/debug is abandoned, you should avoid using it. Use symfony/error-handler instead.
Package symfony/inflector is abandoned, you should avoid using it. Use EnglishInflector from the String component instead.
Generating optimized autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
96 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Run composer recipes at any time to see the status of your Symfony recipes.
Executing script cache:clear [OK]
Executing script assets:install [OK]



# bin/console kimai:update
Kimai updates running ...
=========================
 [OK] Already at the latest version ("DoctrineMigrations\Version20220722125847")
 Rebuilding your cache, please be patient ...
 // Clearing the cache for the prod environment with debug false
 [OK] Cache for the "prod" environment (debug=false) was successfully cleared.
 // Warming up the cache for the prod environment with debug false
 [OK] Cache for the "prod" environment (debug=false) was successfully warmed.
 [OK] Congratulations! Successfully updated Kimai to version 1.22.1

```

**Additional context**
 - Kimai version: 1.23.0

**Comment by kevinpapst** [comment]
Yeah, wrong version number. One again ... how I hate that that manual adjustment problem^^

I need to automate the version increase somehow, this is so annoying.

Thanks for letting me know so asap 👍 you have the latest version anyway.

**Comment by kevinpapst** [comment]
Fixed by #3499 and latest release
